courseware/models.py

```diff
@@ -32,5 +35,8 @@ class OpenEdxApiAuth(TimestampedModel):
     access_token = models.CharField(null=True, max_length=128)
     access_token_expires_on = models.DateTimeField(null=True)
 
+    def __str__(self):
+        return self.user
```

Comment:
This cause my notebook to choke. Changing it to `return f"User {self.user}"` fixed it